test/functional/wallet_descriptor.py

```
@@ -3,6 +3,12 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test descriptor wallet function."""
+import os
+
+try:
+    import sqlite3
+except ImportError:
+    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(ImportError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ImportError)`

(SIM105)